testing/cockroach/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Joe Holden <jwh@zorins.us>
+# Maintainer: Joe Holden <jwh@zorins.us>
+pkgname=cockroach
+pkgver=2.1.5
+_pkgver=v2.1.5
+pkgrel=1
+pkgdesc="The open source, cloud-native SQL database."
+url="https://cockroachlabs.com"
+arch="x86_64"
+license="APL2"
+makedepends="go bash cmake autoconf linux-headers ncurses-dev"
+install="$pkgname.pre-install"
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+subpackages="$pkgname-openrc"
+options="!check"
+source="${pkgname}-${pkgver}.tar.gz::https://binaries.cockroachdb.com/${pkgname}-${_pkgver}.src.tgz
+        cockroach.initd
+        cockroach.confd
+	01_no_githooks.patch"
```

Comment:
Issue with the indentation (should be tab).

Reply:
Indeed it should, fixed